modules/ols-enabling-postgresql-persistence.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+// * lightspeed-docs-main/configure/ols-configuring-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-postgresql-persistence_{context}"]
+= Enabling PostgreSQL persistence
+
+Modify the `OLSConfig` custom resource (CR) file to enable PostgreSQL persistence for the {ols-long} service.
+
+.Prerequisites
+
+* You are logged in to the {ocp-product-title} web console as a user account with permission to create a cluster-scoped CR file, such as a user with the `cluster-admin` role.
+
+* You have installed the {ols-long} Operator.
+
+* You have configured the large language model provider.
+
+.Procedure 
+
+. In the {ocp-product-title} web console, click *Operators* -> *Installed Operators*. 
+
+. Select *All Projects* in the  *Project* list at the top of the screen.
+
+. Click *{ols-long} Operator*.
+
+. Click *OLSConfig*, then click the `cluster` configuration instance in the list.
+
+. Click the *YAML* tab.
+
+. Insert the `spec.ols.storage` YAML code.
++
+.Example `OLSconfig` CR file
+[source,yaml,subs="attributes,verbatim"]
+----
+apiVersion: ols.openshift.io/v1alpha1
+kind: OLSConfig
+metadata:
+  name: cluster
+  namespace: openshift-lightspeed
+spec:
+  llm:
+    providers:
+...
+  ols:
+    storage: {} <1>
+----
+<1> The class depends on the existing instances of the storage class in the cluster. If you leave the storage class empty, {ols-short} uses default values. The persistent volume allocated for the PostgreSQL database is 1 GB in size and uses the storage class of the default cluster. Specify empty braces for the `storage` parameter to use the default values.
++
+If you want to change the `size` and `class` parameters, you can independently specify an explicit value for the parameter. 
++
+[source,yaml,subs="attributes,verbatim"]
+----
+ols:
```

Comment:
technically, this example should have the YAML front matter that you have above. This isn't a blocker, but when you get the chance, please fix it up in a follow up PR.